src/apps/Plotter.jsx

```diff
@@ -0,0 +1,114 @@
+import functionPlot from "function-plot";
+import { useState } from "react";
+import styled from "styled-components";
+
+//------------Styled Components-------------
+
+const Form = styled.form`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  margin: 2px;
+`;
+
+const H3 = styled.h3`
+  color: red;
+`;
+
+//=========================================
+
+let contentsBounds = document.body.getBoundingClientRect();
+let width = 1500;
+let height = 500;
+let ratio = contentsBounds.width / width;
+width *= ratio;
+height *= ratio;
+
+function Renderer() {
+  const [text, changeText] = useState("x^2");
+  const [x_axis, changex] = useState(10);
+  const [y_axis, changey] = useState(5);
+  const [error, changeError] = useState("");
+
+  function handleSubmit(e) {
+    try {
+      e.preventDefault();
+      functionPlot({
+        target: "#instant_apps_root",
+        width,
+        height,
+        xAxis: { domain: [-x_axis, x_axis] },
+        yAxis: { domain: [-y_axis, y_axis] },
+        grid: true,
+        data: [
+          {
+            fn: text,
```

Comment:
We can try removing any extra spaces in the end of the text because the function doesn't work because of it and it's hard to figure out. 
![image](https://user-images.githubusercontent.com/60931421/135667677-245f722b-6041-48ec-a131-69ab1636aa31.png)

Reply:
Sure, looking into this